libcxx/test/libcxx/transitive_includes/cxx03.csv

```diff
@@ -1350,6 +1350,7 @@ memory version
 memory_resource cstddef
 memory_resource cstdint
 memory_resource cstdlib
+memory_resource cstring
```

Comment:
Looks like that I missuggested. `<memory_resource>` starts to include these in C++17, and `<shared_mutex>` starts in C++11 - which is weird.

libc++'s `<shared_mutex>` seems to include internal granularized headers but not provide standard components in C++11 mode.